index.d.ts

```diff
@@ -13,26 +13,56 @@ declare namespace envPaths {
 	export interface Paths {
 		/**
 		Directory for data files.
+
+		Example locations:
+
+		- macOS: `~/Library/Application Support/MyApp-nodejs`
+		- Windows: `%LOCALAPPDATA%\MyApp-nodejs\Data` (for example, `C:\Users\USERNAME\AppData\Local\MyApp-nodejs\Data`)
+		- Linux: `~/.local/share/MyApp-nodejs` (or `$XDG_DATA_HOME/MyApp-nodejs`)
 		*/
 		readonly data: string;
 
 		/**
 		Directory for data files.
+
+		Example locations:
+
+		- macOS: `~/Library/Preferences/MyApp-nodejs`
+		- Windows: `%APPDATA%\MyApp-nodejs\Config` (for example, `C:\Users\USERNAME\AppData\Roaming\MyApp-nodejs\Config`)
+		- Linux: `~/.config/MyApp-nodejs` (or `$XDG_CONFIG_HOME/MyApp-nodejs`)
 		*/
 		readonly config: string;
 
 		/**
 		Directory for non-essential data files.
+
+		Example locations:
+
+		- macOS: `~/Library/Caches/MyApp-nodejs`
+		- Windows: `%LOCALAPPDATA%\MyApp-nodejs\Cache` (for example, `C:\Users\USERNAME\AppData\Local\MyApp-nodejs\Cache`)
+		- Linux: `~/.cache/MyApp-nodejs` (or `$XDG_CACHE_HOME/MyApp-nodejs`)
 		*/
 		readonly cache: string;
 
 		/**
 		Directory for log files.
+
+		Example locations:
+
+		- macOS: `~/Library/Logs/MyApp-nodejs`
+		- Windows: `%LOCALAPPDATA%\MyApp-nodejs\Log` (for example, `C:\Users\USERNAME\AppData\Local\MyApp-nodejs\Log`)
+		- Linux: `~/.local/state/MyApp-nodejs` (or `$XDG_STATE_HOME/MyApp-nodejs`)
 		*/
 		readonly log: string;
 
 		/**
 		Directory for temporary files.
+
+		Example locations (with the default `nodejs` [suffix](#suffix)):
```

Comment:
The Markdown link should not be used here.

Reply:
Fixed in 0b20ba6.

Reply:
I think you misunderstood. The text should still be there. It should just not be a link.

Reply:
I've added the text back (for all properties) in cbefe18187ef5b642624ded5f57bfb2ceb9c2eeb.